accounts/templates/logout.html

```diff
@@ -0,0 +1,7 @@
+<a id="openstax-logout" 
+   name="openstax" 
+   href="{% url 'logout' %}" 
+   id="openstax-logout">
+   <input type="button" value= "Logout you will still be logged into the Openstax OAuth2 Provider"/>
```

Comment:
That's what you said before - but that's not what is happening for me. Regardless, I think the wording is unclear, it should just say 'Logout'. The way it is currently working for me seems like it's working appropriately, it's just not what the button says. Still not sure where this is getting rendered though.